docs/usage/self-hosted-configuration.md

```diff
@@ -200,6 +200,13 @@ It will also override any settings in `packageRules`.
 ## forkMode
 
 You probably have no need for this option - it is an experimental setting for the Renovate hosted GitHub App.
+However, if this is set to true then Renovate will clone the repository before working on the clone and creating Pull Requests on the parent (GitHub only).
```

Comment:
```suggestion
However, if this is set to `true` then Renovate will clone the repository before working on the clone and creating Pull Requests on the parent (GitHub only).
```

Let's put `true` in monospaced font.

Also it's really confusing to read: "Renovate will clone the repository before working on the clone"...
I don't know what the behavior is, but the way its worded now is not helping me understand whats happening.

The way I understand Renovate is that it _always_ clones the repo before doing stuff???

Reply:
Correct, wording is misleading, I meant fork instead of clone. Let me rephrase.

Reply:
> Correct, wording is misleading, I meant fork instead of clone. Let me rephrase.

I thought some wires got crossed somewhere. 😄 
I'll let you rephrase this. 😉

Reply:
I have updated the wording, does it make more sense now?